File: Task 1/Java/app/src/main/java/dev/ogabek/java/InformationActivity.java
package dev.ogabek.java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class InformationActivity extends AppCompatActivity {

    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_information);

        initViews();
        getInformation();

    }

    void initViews() {
        textView = findViewById(R.id.textView);
    }

    void getInformation() {
        String name = getIntent().getStringExtra("name");
        String surname = getIntent().getStringExtra("surname");
        textView.setText(name + " " + surname);
    }

}

File: Task 1/Kotlin/app/src/main/java/dev/ogabek/kotlin/InformationActivity.kt
package dev.ogabek.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class InformationActivity : AppCompatActivity() {

    private lateinit var txt: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_information)

        initViews()
        getInfo()

    }

    private fun initViews() {
        txt = findViewById(R.id.name)
    }

    private fun getInfo() {
        val name = intent.getStringExtra("name")
        val surname = intent.getStringExtra("surname")
        txt.text = "$name $surname"
    }

}